Jenkinsfile.infra.ci.jenkins.io

```diff
@@ -1,10 +1 @@
-@Library('pipeline-library@pull/459/head') _
-
-// This pipeline runs on infra.ci.jenkins.io (private) to build, test and deploy the Docker image
-buildDockerAndPublishImage('ath', [
-  automaticSemanticVersioning: false,
-  gitCredentials: 'jenkinsci-ath-ghapp', // Only available in infra.ci.jenkins.io
-  dockerfile: 'src/main/resources/ath-container/Dockerfile',
-  imageDir: 'src/main/resources/ath-container/',
-  registryNamespace: 'jenkins',
-])
+echo 'not needed'
```

Comment:
This will disable the build + deploy of the Docker image.

> Additionally, I disabled the pipeline for builds on infra.ci, because of several hadolint violations.

WDYT about fixing the hadolint violations? We are using it for safety reasons and there are ways to opt out.

Reply:
(I'm checking the violations to give you actionnables)

Reply:
@NotMyFault you should be able to see the violations in https://infra.ci.jenkins.io/job/docker-jobs/job/acceptance-test-harness/job/PR-878/1/jenkins-ath-hadolint-1664352940229/

Reply:
> This will disable the build + deploy of the Docker image.

I always comment out the build on infra.ci when I run core PR tests, because I just need ATH tests.

I was checking if something requires actual attention, aside from possible test failures on the master branch. The commented version is not intended to be merged though, hadolint is still up to be addressed.

Reply:
There we go, local `hadolint` is clean of errors for the docker file 👍🏻